selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -255,7 +253,7 @@ void WifiManager::deactivate_connections(const QString &ssid) {
       if (Ssid == ssid) {
         QDBusInterface nm2(nm_service, nm_path, nm_iface, bus);
         nm2.setTimeout(dbus_timeout);
-        nm2.call("DeactivateConnection", QVariant::fromValue(active_connection_raw));// TODO change to disconnect
```

Comment:
are you sure that's what the comment meant?

Reply:
I'm pretty sure, `DeactivateConnection` disconnects from the network